cabby/data/wikidata/query.py

```diff
@@ -0,0 +1,192 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Library to support Wikidata areal queries.'''
+
+import sys
+from SPARQLWrapper import SPARQLWrapper, JSON
+from typing import Dict, Tuple, Sequence, Text
+
+
+def get_wikidata(map_name: Text) -> Dict:
+    '''Get Wikidata items for a specific area. 
+    Arguments:
+        map_name(Text): The area to query the Wikidata items.
+    Returns:
+        The Wikidata items found in the area.
+    '''
+
+    assert map_name == "Manhattan" or map_name == "Pittsburgh"
+    if map_name == "Pittsburgh":
+        query = """SELECT ?place ?placeLabel
+                    (GROUP_CONCAT(DISTINCT?location;separator=", ") AS ?point)
+                    WHERE 
+                    {
+                    {
+                        ?place wdt:P31 ?instance.
+                        SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
+                        SERVICE wikibase:box {
+                        ?place wdt:P625 ?location .
+                        bd:serviceParam wikibase:cornerWest "Point(-80.035,40.425)"^^geo:wktLiteral .
+                        bd:serviceParam wikibase:cornerEast "Point(-79.930,40.460)"^^geo:wktLiteral .
+                        }
+                    }
+                    FILTER (?instance  not in
+                    (wd:Q34442,wd:Q12042110,wd:Q124757,wd:Q79007,wd:Q18340514,wd:Q537127,wd:Q1311958,wd:Q124757,
+                    wd:Q25917154,  wd:Q1243306, wd:Q1570262, wd:Q811683,
+                    wd:Q744913          ) )
+                    }
+                    GROUP BY ?place ?placeLabel
+                """
```

Comment:
Pull this out of the function and set it as a global variable _PITTSBURGH_QUERY (same for manhattan).

Reply:
done